server/sonar-server/src/main/java/org/sonar/server/es/RecoveryIndexer.java

```diff
@@ -30,6 +30,7 @@
 import org.apache.commons.lang.math.RandomUtils;
 import org.sonar.api.Startable;
 import org.sonar.api.config.Configuration;
+import org.sonar.api.config.Settings;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.sonar.api.config.Settings'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)